diff -ub phpBB3/includes/bbcode.php phpBB3-new/includes/bbcode.php
--- phpBB3/includes/bbcode.php	2018-11-11 01:14:16.000000000 -0800
+++ phpBB3-new/includes/bbcode.php	2018-11-18 14:02:48.000000000 -0800
@@ -35,10 +35,19 @@
 
 	/**
 	* Constructor
-	* Init bbcode cache entries if bitfield is specified
 	*/
 	function __construct($bitfield = '')
 	{
+		$this->bbcode_set_bitfield($bitfield);
+	}
+
+	/**
+	* Init bbcode cache entries if bitfield is specified
+	*
+	* @param	string	$bbcode_bitfield	The bbcode bitfield
+	*/
+	function bbcode_set_bitfield($bitfield = '')
+	{
 		if ($bitfield)
 		{
 			$this->bbcode_bitfield = $bitfield;
diff -ub phpBB3/includes/functions_content.php phpBB3-new/includes/functions_content.php
--- phpBB3/includes/functions_content.php	2018-11-11 01:14:16.000000000 -0800
+++ phpBB3-new/includes/functions_content.php	2018-11-18 14:03:42.000000000 -0800
@@ -627,7 +627,7 @@
 			}
 			else
 			{
-				$bbcode->bbcode($bitfield);
+				$bbcode->bbcode_set_bitfield($bitfield);
 			}
 
 			$bbcode->bbcode_second_pass($text, $uid);
